### PR TITLE
Add option to display custom fields under "Additional Information" tab on PDP

### PR DESCRIPTION
**Pull request by sacr3dc0w**
#### What?

Adds an option to display custom fields under a "Additional Information" tab.

#### Tickets / Documentation

- #1065 
- #1295 

#### Screenshots (if appropriate)

**False, default**
01 false default

**False, default - tab region**
02 false tab region

**True**
03 true

**True, tab region - closed**
04 true tab closed

**True, tab region - open**
05 true tab open

**True, tab region - open, mobile**
true mobile

**True, tab region - closed**
06 theme editor

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -1285,6 +1285,20 @@
         "force_reload": true,
         "id": "show_product_details_tabs"
       },
+      {
```
Comment (junedkazi) [inline schema.json]: @bigcommerce/merc-team

File: templates/components/products/description-tabs.html [hunk 2]
```diff
@@ -7,6 +7,11 @@
             <a class="tab-title" href="#tab-warranty">{{lang 'products.warranty'}}</a>
         </li>
     {{/if}}
+    {{#all product.custom_fields theme_settings.show_custom_fields_tabs}}
+        <li class="tab">
+            <a class="tab-title" href="#tab-specs">{{theme_settings.pdp-custom-fields-tab-label}}</a>
```
Comment (junedkazi) [inline templates/components/products/description-tabs.html]: Can we build the href based on the label so it matches what we are actually calling the tab.

File: templates/components/products/description-tabs.html [hunk 3]
```diff
@@ -22,6 +27,16 @@
            {{{product.warranty}}}
        </div>
    {{/if}}
+   {{#all product.custom_fields theme_settings.show_custom_fields_tabs}}
+      <div class="tab-content" id="tab-specs">
```
Comment (junedkazi) [inline templates/components/products/description-tabs.html]: Can we build the href based on the label so it matches what we are actually calling the tab.

Comment (sacr3dc0w) [reply templates/components/products/description-tabs.html]: Screen Shot 2019-04-22 at 4 23 39 PM

Comment (sacr3dc0w) [reply templates/components/products/description-tabs.html]: Screen Shot 2019-04-22 at 4 27 24 PM

File: templates/components/products/description-tabs.html (hunk 2 above)
Comment (sacr3dc0w) [reply templates/components/products/description-tabs.html]: Screen Shot 2019-04-22 at 4 23 39 PM

**Comment by sacr3dc0w** [comment]
@junedkazi Ran the string through `{{sanitize}}` in case merchants try to include HTML in the label, then `{{lowercase}}` and `{{dashcase}}` for cleaner output.

File: config.json [hunk 4]
```diff
@@ -304,6 +305,7 @@
     "pdp-sale-price-label": "Now:",
     "pdp-non-sale-price-label": "Was:",
     "pdp-retail-price-label": "MSRP:",
+    "pdp-custom-fields-tab-label": "Specifications",
```
Comment (junedkazi) [inline config.json]: Can we also change the label to be more generic like "Additional Information" ? 🍹

**Pull request merged**